docs/vocascan-frontend/installation/traefik.md

```diff
@@ -1,3 +1,53 @@
 # Traefik
 
-## Coming soon
+## Prerequisites
+
+This guide assumes, that you already installed docker and docker-compose as it is described in the
+[install server via docker guide](vocascan-server/installation/docker).
+
+## 1. Adjust docker-compose.yml
+
+Add the following service to your existing `docker-compose.yml` file and update the domains to your existing one.
+
+?> This step assumes, that you already have a running Traefik reverse proxy, with all it's config and Docker-Compose
+files, running, to just add the next config snippet. If you need a tutorial on how to do this, follow our
+["how to set up vocascan-server with Traefik"](vocascan-server/installation/traefik) Tutorial (if you don't need the
+server part, just delete it).
+
+```yml
+---
+frontend:
+  image: vocascan/frontend:latest
+  restart: always
+  tty: true
+  environment:
+    VOCASCAN_BASE_URL: "https://your-api-domain.com"
+  labels:
+    - "traefik.http.routers.frontend.rule=Host(`your-domain.com`)"
+    - "traefik.http.routers.frontend.tls=true"
+    - "traefik.http.routers.frontend.tls.certresolver=lets-encrypt"
+    - "traefik.http.services.frontend.loadbalancer.server.port=80"
+  networks:
+    - internal
```

Comment:
why do you added internal?

Reply:
I made those configs after tutorials. In those you give everything, that you put everything that should available from outside in the `internal` and `web` network

Reply:
Ok. But can you please remove the internal network (and also in production) as the nginx server doesnt neet to connect to the database.

Reply:
I need the internal network for the server to find the database. Shouldn't be a security issue, as `external` is set to `false`

Reply:
Yeah, but you don't need the internet network on the Frontend. That's my point.

Reply:
You are right. I just removed it